wurst/_handles/Unit.wurst

```diff
@@ -151,6 +151,9 @@ public function unit.getMoveSpeed() returns real
 public function unit.getName() returns string
 	return GetUnitName(this)
 
+public function unit.getProperName() returns string
```

Comment:
What's the behavior for non-heroes?

Reply:
Simply returns null.

Reply:
Is it documented in lep/jassdoc? Can you document it here? Is it right for this to be an extension function of `unit`?

Reply:
1) Jassdoc looks like it's hopelessly outdated. Someone needs to bring over all the new natives introduced into the latest patches there, but I am personally not up to the task. Just having one and not others would be weird.

2) Other hero-related functions are exposed as extension funcs on top of `unit`, so I don't see any consistency issues here.

Reply:
1. I disagree about it being weird to have one but not others. If you don't have time to make a MR, can you at least raise an issue specifying that unit.getProperName returns null for non-hero?

2. Ok

Reply:
> Jassdoc looks like it's hopelessly outdated.

Why?
I don't think writing documentation about broken new natives, which get changed by pretty much any successive patch, has high priority.
Is there anything else outdated? I think jassdoc actually contains docs which cannot be found anywhere else. And unlike questionable pinned threads that only cover a small amount of natives in depth, this is an amazing, very structured and consistent overview.
And in its form it can be more easily used.

Reply:
I think the new natives are going to more or less stay in their current form from now on, apart from minor fixes to them, and it would be safe to include them and note any outstanding bugs, because some of them are quite useful.

Either way, I sent a PR to lep to update that particular function. I just thought it was one of the new natives - which, turns out, it isn't.